font-inspector.html

```diff
@@ -162,21 +164,31 @@ <h1>Free Software</h1>
 
 function displayNames(names) {
     var html = '';
-    properties = sortKeys(names);
-    for (var i = 0; i < properties.length; i++) {
-        var property = properties[i];
-        html += '<dt>'+escapeHtml(property)+'</dt><dd>';
-        var translations = names[property];
-        var langs = sortKeys(translations);
-        for (var j = 0; j < langs.length; j++) {
-            var lang = langs[j];
-            var esclang = escapeHtml(lang);
-            html += '<span class="langtag">' + esclang
-                + '</span> <span class="langname" lang=' + esclang + '>'
-                + escapeHtml(translations[lang]) + '</span> ';
+    ['unicode', 'macintosh', 'windows'].forEach( platform => {
+        let platformNames = names?.[platform];
```

Comment:
This still has let as well as optional chaining, should probably be:
```js
var platformNames = (names?names.platform:null);
```

Reply:
fixed the comment, there was a misunderstanding.

Reply:
Couldn't amend because I'm currently on mobile and limited to the web editor